library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/impl/QuiltBuiltinResourcePackProfile.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.resource.loader.impl;
+
+import com.mojang.logging.LogUtils;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.resource.ResourcePackCompatibility;
+import net.minecraft.resource.ResourcePackProfile;
+import net.minecraft.resource.ResourcePackSource;
+import net.minecraft.resource.metadata.PackResourceMetadata;
+import net.minecraft.text.LiteralText;
+import net.minecraft.text.Text;
+import net.minecraft.text.TranslatableText;
+import net.minecraft.util.Formatting;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.ApiStatus;
+import org.quiltmc.qsl.resource.loader.api.ResourcePackActivationType;
+import org.slf4j.Logger;
+
+import java.io.IOException;
+
+@ApiStatus.Internal
+public final class QuiltBuiltinResourcePackProfile extends ResourcePackProfile {
+	private static final Logger LOGGER = LogUtils.getLogger();
```

Comment:
Who does this log as? Does it somehow determine it was created in the `QuiltBuiltinResourcePackProfile`?

Reply:
Yup, this is a shortcut method by Mojang that restore the getLogger behavior from log4j.